src/Symfony/Component/Config/Tests/Builder/Fixtures/ScalarNormalizedTypes/Symfony/Config/ScalarNormalizedTypesConfig.php

```diff
@@ -128,6 +130,46 @@ public function nested(array $value = []): \Symfony\Config\ScalarNormalizedTypes
         return $this->nested;
     }
 
+    /*
+     * @param array{
+     *     simple_array?: array<array-key, mixed>,
+     *     keyed_array?: array<array-key, mixed>,
+     *     object?: array{
+     *         enabled?: bool,
+     *         date_format?: string|int|float|bool,
+     *         remove_used_context_fields?: bool,
+     *     },
+     *     list_object: array<array-key, mixed>,
+     *     keyed_list_object?: array<array-key, mixed>,
+     *     nested?: array{
+     *         nested_object?: array{
+     *             enabled?: bool,
+     *         },
+     *         nested_list_object?: array<array-key, mixed>,
+     *     },
+     * } $config
+     */
+    public function configure(#[ArrayShape([
```

Comment:
let's ensure each argument start on its own line with a starting comment to help the reader:
```suggestion
    public function configure(
        // config:
        #[ArrayShape([
```